Package.swift

```diff
@@ -5,13 +5,15 @@ import Foundation
 
 let buildDynamic = ProcessInfo.processInfo.environment["NODE_SWIFT_BUILD_DYNAMIC"] == "1"
 let enableEvolution = ProcessInfo.processInfo.environment["NODE_SWIFT_ENABLE_EVOLUTION"] == "1"
+let targetMacVersion = ProcessInfo.processInfo.environment["NODE_SWIFT_TARGET_MAC_VERSION"]!
```

Comment:
Instead of force unwrapping we should probably provide a default value 💅

Reply:
perhaps I wouldn't even use an env variable... I think we always want `.macOS(.v10_15),` ?

Reply:
It will be always there taken from the main (push-notifications) package platform value, but I agree it's better to have default fallback for safety.